### PR TITLE
Add help menu link for filing issues

### PR DESCRIPTION
**Pull request by Lukewh**
Fixes https://github.com/juju/juju-gui/issues/2462

### CODE REVIEW

File: jujugui/static/gui/src/app/components/header-help/header-help.js [hunk 1]
```diff
@@ -56,11 +58,33 @@ YUI.add('header-help', function() {
     },
 
     /**
+      Generate a link to issues based on whether the user is logged in
+      and in gisf
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: Final dot.

File: jujugui/static/gui/src/app/components/header-help/header-help.js [hunk 2]
```diff
@@ -56,11 +58,33 @@ YUI.add('header-help', function() {
...
+
+      @method _generateIssuesLink
+     */
+    _generateIssuesLink: function() {
+      let link = 'https://github.com/juju/juju-gui/issues';
+
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: Unnecessary newline.

File: jujugui/static/gui/src/app/components/header-help/header-help.js [hunk 3]
```diff
@@ -56,11 +58,33 @@ YUI.add('header-help', function() {
...
+      if (this.props.gisf && this.props.user) {
+        link = 'https://jujucharms.com/issues';
+      }
+
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: Ditto.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6804/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6807/

**Comment by Lukewh** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**